Makefile

```diff
@@ -75,6 +75,9 @@ testinsights:
 unit: generate testinsights
 	go test ./... -coverprofile=coverage.out -covermode=atomic -json | ./testinsights
 
+sec:
+	./hack/verify/validate-sec.sh
```

Comment:
Can you add this to verify target instead? this would trigger it to run on CI and "one command to verify them all" locally? :)

Reply:
Sure! done :)